pulsar-functions/runtime/src/main/java/org/apache/pulsar/functions/runtime/LocalRunner.java

```diff
@@ -108,6 +108,8 @@ void start() throws Exception {
                     }
                     classLoader = loadJar(file);
                 }
+            } else if (functionConfig.getRuntime() == FunctionConfig.Runtime.GO) {
+                userCodeFile = functionConfig.getGo();
```

Comment:
Can we also check for python runtime here and throw an exception if its not Java, Go, or python